appchain-registry/src/storage_key.rs

```diff
@@ -19,7 +18,6 @@ impl StorageKey {
         match self {
             StorageKey::AppchainIds => "i".to_string(),
             StorageKey::AppchainBasedatas => "a".to_string(),
-            StorageKey::ContractCode => "cc".to_string(),
             StorageKey::UpvoteDeposits => "u".to_string(),
             StorageKey::DownvoteDeposits => "d".to_string(),
             StorageKey::RegistrySettings => "rs".to_string(),
```

Comment:
We could have implemented BorshStorageKey for enum StorageKey,then we can use StorageKey item as Storage prefix directly. eg from ref-exchange contract:
``` Rust
#[derive(BorshStorageKey, BorshSerialize)]
pub(crate) enum StorageKey {
    Pools,
    Accounts,
    Shares { pool_id: u32 },
    Whitelist,
    Guardian,
    AccountTokens {account_id: AccountId},
}
```

``` Rust
   Self {
            owner_id: owner_id.as_ref().clone(),
            exchange_fee,
            referral_fee,
            pools: Vector::new(StorageKey::Pools),
            accounts: LookupMap::new(StorageKey::Accounts),
            whitelisted_tokens: UnorderedSet::new(StorageKey::Whitelist),
            guardians: UnorderedSet::new(StorageKey::Guardian),
            state: RunningState::Running,
        }
```
``` rust
 tokens: UnorderedMap::new(StorageKey::AccountTokens {
                account_id: account_id.clone(),
            }),
```